static/include/robot/apis/generated/robot.md

```diff
@@ -0,0 +1,760 @@
+const status = await machine.getStatus();
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### StopAll
+
+Cancel all current and outstanding operations for the machine and stop all actuators and movement.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `extra` (Mapping[[str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str), Any]) (required): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- None.
+
+**Example:**
+
+```python {class="line-numbers linkable-line-numbers"}
+# Cancel all current and outstanding operations for the robot and stop all actuators and movement.
+await robot.stop_all()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.stop_all).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context#Context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map[string]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
+
+**Returns:**
+
+- [(error)](https://pkg.go.dev/builtin#error): An error, if one occurred.
+
+**Example:**
+
+```go {class="line-numbers linkable-line-numbers"}
+// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
+err := machine.StopAll(ctx)
+```
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/robot#Robot).
+
+{{% /tab %}}
+{{% tab name="TypeScript" %}}
+
+**Parameters:**
+
+- None
+
+**Returns:**
+
+- None
+
+For more information, see the [TypeScript SDK Docs](https://ts.viam.dev/classes/RobotClient.html#stopAll).
+
+```typescript {class="line-numbers linkable-line-numbers"}
+// Cancel all current and outstanding operations for the machine and stop all actuators and movement.
+await machine.stopAll();
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Log
+
+Create a LogEntry object from the log to send to the RDK over gRPC.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `name` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The logger’s name.
+- `level` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The level of the log.
+- `time` ([datetime.datetime](https://docs.python.org/3/library/datetime.html)) (required): The log creation time.
+- `log` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The log message.
+- `stack` ([str](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str)) (required): The stack information of the log.
+
+**Returns:**
+
+- None.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.log).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetCloudMetadata
```

Comment:
missing python code example

Reply:
There [isn't one](https://python.viam.dev/autoapi/viam/robot/client/index.html#viam.robot.client.RobotClient.get_cloud_metadata). I've added manually from what is live in docs currently.